src/adapters/node.ts

```
@@ -48,27 +48,36 @@ class NodeServer implements Server {
 
-    gracefulShutdownPlugin(this);
-
     const fetchHandler = (this.fetch = wrapFetch(this));
 
+    // Detect running in srvx loader
+    const loader = (globalThis as any).__srvxLoader__ as
+      | ((handler: ServerHandler) => void)
+      | undefined;
+    if (loader) {
+      loader(fetchHandler);
+      return this;
```

Biome (2.3.14)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)